src/test/java/edu/hawaii/its/api/groupings/GroupingAddResultTest.java

```diff
@@ -26,11 +26,11 @@
 @SpringBootTest(classes = { SpringBootWebApplication.class })
 public class GroupingAddResultTest {
 
-    @Value("${groupings.api.test.uh-usernames}")
-    private List<String> TEST_USERNAMES;
+    @Value("${groupings.api.test.uh-uids}")
+    private List<String> TEST_UIDS;
```

Comment:
Make sure to update all places with `groupings.api.test.uh-uids` to `groupings.api.test.uids`, thanks.